### PR TITLE
Import style.css if present in configuration

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

**Comment by mattwigway** [comment]
@trevorgerhardt LGTM, although I'm wondering if it would make sense to have paths in the CSS that get base64ified relative to the config directory rather than the taui directory.

**Comment by trevorgerhardt** [comment]
@mattwigway I've been debating how to implement that. 

Right now I'm appending the CSS right into the entry file. That allows us to override CSS rules extremely easily since the configuration comes AFTER the entry. We can just right the same rule twice and not do an `!important` or more specific rule.

Another strategy would be to prepend an `@import` statement to that configuration file which would easily allow for local paths in the stylesheet. This wouldn't allow overriding rules as easily and would require CSS variables to be added into the base stylesheet. This would make the "hooks" more obvious for someone writing a configuration file, but would take a bit more work.

It seems like the CORRECT way to implement this. Since it will fix the image paths without any other magic, I think I'm going to do it that way.

**Comment by mattwigway** [comment]
We could also have a CSS variable that's defined as the path to the custom configuration file. So you'd say `background-image: url(--CONFIGURATION-PATH/logo.png)`. That would be portable.

**Comment by trevorgerhardt** [comment]
@evansiroky how does the latest update look?

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Auto-populate your redux store with this configuration data instead of setting d
 
 ### `style.css`
 
-Add a stylesheet that gets appended at the end of your entry stylesheet. This allows you to override styles for specific deployments and use [custom CSS properties](http://cssnext.io/features/#custom-properties-var).
+Add a stylesheet that gets `@import`ed at the beginning of your entry stylesheet. This allows you to override styles for specific deployments and use [custom CSS properties](http://cssnext.io/features/#custom-properties-var). Useful for configuration specific images and colors.
```
Comment (landonreed) [inline README.md]: @trevorgerhardt I think you forgot to include `style.css` in the list of config files above: https://github.com/conveyal/mastarm/pull/146/commits/d1bdc4ff274ae6eca97264e0689669cd781f69f7#diff-04c6e90faac2675aa89e2176d2eec7d8R32

**Comment by evansiroky** [comment]
The introduction in the configuration section should get updated.  Maybe the sentence with the list of files can be removed.

File: README.md (hunk 1 above)
Comment (trevorgerhardt) [inline README.md]: @landonreed good catch 👌

**Comment by trevorgerhardt** [comment]
@evansiroky & @landonreed updated 👌

**Pull request merged**